tests/Integration/inc/Engine/CriticalPath/RestTestCase.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace WP_Rocket\Tests\Integration\inc\Engine\CriticalPath;
+
+use WP_Rest_Request;
+use WP_REST_Response;
+use WP_REST_Server;
+use WP_Rocket\Tests\Integration\FilesystemTestCase;
+
+abstract class RestTestCase extends FilesystemTestCase {
+	/**
+	 * Instance of the WordPress REST Server.
+	 * @var WP_REST_Server
+	 */
+	protected $server;
+
+	/**
+	 * Setup the WP REST API Server.
+	 */
+	public function setUp() {
+		parent::setUp();
+		/**
+		 * @var WP_REST_Server $wp_rest_server
+		 */
+		global $wp_rest_server;
+		$this->server = $wp_rest_server = new WP_REST_Server;
+		do_action( 'rest_api_init' );
```

Comment:
Is there a reason not to use `rest_get_server()`?

```suggestion
		$this->server = rest_get_server();
```

Reply:
@Screenfeed This will be removed once we merge PR 20 https://github.com/wp-media/phpunit/pull/20

Reply:
Yes, sorry, I reviewed this before https://github.com/wp-media/phpunit/pull/20.